src/redux/actions/actions.js

```diff
@@ -1,26 +1,47 @@
 import * as actionTypes from './actionTypes';
 
-export function GetPosts() {
+export function getUsersAndPosts() {
   return function(dispatch) {
-    fetch('https://jsonplaceholder.typicode.com/posts')
-      .then(response => response.json())
-      .then(json => {
-        dispatch({
-          type: actionTypes.SAVE_POSTS,
-          postsData: json.slice(-10).sort((a, b) => a.id < b.id),
-        });
-      });
+    const usersData = getUsers();
+    const postsData = getPosts();
+    // eslint-disable-next-line no-undef

```

Comment:
Why you need to disable this rule here?

Reply:
eslint shows an error "Promise is not defined"

Reply:
You should change eslint config so it knows that `Promise` is always available

Reply:
Lazy way isn't better way